src/App.js

```diff
@@ -56,6 +61,25 @@ export default class App extends Component {
       : this.addCard(newCard)
   }
 
+  toggleShowAnswer() {
```

Comment:
This method and `handleCarouselControls` should be implemented within `Practice`.